packages/module/src/components/VncConsole/VncConsole.tsx

```diff
@@ -10,11 +9,24 @@ import RFB from '@novnc/novnc/core/rfb';
 import { VncActions } from './VncActions';
 import { constants } from '../common/constants';
 
-import styles from '@patternfly/react-styles/css/components/Consoles/VncConsole';
-import '@patternfly/react-styles/css/components/Consoles/VncConsole.css';
+import clsx from 'clsx';
+import { createUseStyles } from 'react-jss';
 
 const { CONNECTED, CONNECTING, DISCONNECTED } = constants;
 
+const useStyles = createUseStyles({
+  consoleVnc: {
+    gridArea: 'main'
+  },
+  consoleActionsVnc: {
+    gridArea: 'actions-extra',
+    display: 'flex',
```

Comment:
I wonder if we can use a flex layout here instead of all these flex related css overrides

Reply:
Created https://github.com/patternfly/react-console/issues/62